dnssec.go

```diff
@@ -543,35 +545,37 @@ func (k *DNSKEY) publicKeyRSA() *rsa.PublicKey {
 		keyoff = 3
 	}
 
-	if explen > 4 || explen == 0 || keybuf[keyoff] == 0 {
+	pubkey := new(rsa.PublicKey)
```

Comment:
why has this been moved?

Reply:
Ditto. No need for this to be moved up here. It would make sense for it to be at the end, but there's no real need to move it anyway.